library/src/main/java/com/chuckerteam/chucker/api/ChuckerInterceptor.kt

```diff
@@ -161,6 +170,17 @@ class ChuckerInterceptor @JvmOverloads constructor(
         return response.body()!!.source()
     }
 
+    private fun findOperationName(content: JsonObject): String? {
+        val key = "operationName"
+        if (content.has(key)) return content.get(key).asString
+
+        return if (content.has("queryContainer")) {
```

Comment:
Hey @rnitame 
Apparently here there is a problem. GraphQL request should have the `operationName` at the top level and should not be wrapped inside a `queryContainer` object:

![Screenshot_1565099160](https://user-images.githubusercontent.com/3001957/62545092-4e822000-b861-11e9-8d8c-b3e606e2636b.png)

This is an invalid GraphQL request.

The problem is caused by the initialisation of the retrofit interface:
```
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("https://graphql.org")
                .addConverterFactory(GsonConverterFactory.create())
                .client(client)
                .build();
```
Although we could fix it by using the Converter factory provided by retrofit-graphql, I think it's better if we change approach here.

Can I ask you to:

a) Don't use `retrofit-graphql`. Is yet another dependency that we probably don't need.
b) Just create a simple `data class` and do a GraphQL request without external library (as described here: https://graphql.org/learn/serving-over-http/#post-request)
c) Remove all the .graphql file?